### PR TITLE
autoflake: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — COMMENTED** [review]
I think it would be better with this small changes

File: pages.es/common/autoflake.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# autoflake
+
+> Una herramienta para eliminar importaciones y variables no utilizadas del código Python.
+> Más información: <https://github.com/myint/autoflake>.
+
+- Elimina las variables no utilizadas de un único archivo y muestra la diferencia:
+
+`autoflake --remove-unused-variables {{archivo.py}}`
```
Comment (Magrid0) [inline pages.es/common/autoflake.md:8]:
```suggestion
`autoflake --remove-unused-variables {{ruta\al\archivo.py}}`
```

File: pages.es/common/autoflake.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Elimina las importaciones no utilizadas de varios archivos y muestra las diferencias:
+
+`autoflake --remove-all-unused-imports {{archivo1.py}} {{archivo2.py}} {{archivo3.py}}`
```
Comment (Magrid0) [inline pages.es/common/autoflake.md:12]:
```suggestion
`autoflake --remove-all-unused-imports {{ruta\al\archivo1.py}} {{ruta\al\archivo2.py}} {{ruta\al\archivo3.py}}`
```

File: pages.es/common/autoflake.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Elimina variables no utilizadas de un archivo, sobrescribiendo el archivo:
+
+`autoflake --remove-unused-variables --in-place {{archivo.py}}`
```
Comment (Magrid0) [inline pages.es/common/autoflake.md:16]:
```suggestion
`autoflake --remove-unused-variables --in-place {{ruta\al\archivo.py}}`
```

**Review by Magrid0 — COMMENTED** [review]
I think it would be better with this small changes

File: pages.es/common/autoflake.md (hunk 2 above)
Comment (kbdharun) [reply pages.es/common/autoflake.md:12]: Good suggestion, I will update the same in the English page.

Comment (kbdharun) [reply pages.es/common/autoflake.md:12]: Updated your current suggestion to tldr syntax.

File: pages.es/common/autoflake.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# autoflake
+
+> Una herramienta para eliminar importaciones y variables no utilizadas del código Python.
+> Más información: <https://github.com/myint/autoflake>.
+
+- Elimina las variables no utilizadas de un único archivo y muestra la diferencia:
+
+`autoflake --remove-unused-variables {{ruta\al\archivo.py}}`
```
Comment (kbdharun) [inline pages.es/common/autoflake.md:8]:
```suggestion
`autoflake --remove-unused-variables {{ruta/al/archivo.py}}`
```

File: pages.es/common/autoflake.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# autoflake
+
+> Una herramienta para eliminar importaciones y variables no utilizadas del código Python.
```
Comment (realFranco) [inline pages.es/common/autoflake.md:3]: Hello, thank you for the contribution, I will add some suggestions:

```suggestion
> Herramienta para eliminar importaciones y variables no utilizadas en archivos con extensión y sintáxis válida en Python.
```

File: pages.es/common/autoflake.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> Más información: <https://github.com/myint/autoflake>.
+
+- Elimina las variables no utilizadas de un único archivo y muestra la diferencia:
```
Comment (realFranco) [inline pages.es/common/autoflake.md:6]:
```suggestion
- Elimina variables no utilizadas de un único archivo y muestra las diferencias:
```

File: pages.es/common/autoflake.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`autoflake --remove-unused-variables {{ruta\al\archivo.py}}`
+
+- Elimina las importaciones no utilizadas de varios archivos y muestra las diferencias:
```
Comment (realFranco) [inline pages.es/common/autoflake.md:10]:
```suggestion
- Elimina importaciones no utilizadas de varios archivos y muestra las diferencias de cada uno de ellos:
```

File: pages.es/common/autoflake.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`autoflake --remove-all-unused-imports {{archivo1.py}} {{archivo2.py}} {{archivo3.py}}`
+
+- Elimina variables no utilizadas de un archivo, sobrescribiendo el archivo:
```
Comment (realFranco) [inline pages.es/common/autoflake.md:14]:
```suggestion
- Elimina variables no utilizadas de un archivo, sobrescribiendo el mismo:
```

File: pages.es/common/autoflake.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`autoflake --remove-unused-variables --in-place {{archivo.py}}`
+
+- Elimina las variables no utilizadas recursivamente de todos los archivos en un directorio, sobrescribiendo cada archivo:
```
Comment (realFranco) [inline pages.es/common/autoflake.md:18]:
```suggestion
- Elimina variables no utilizadas recursivamente de todos los archivos en un directorio, sobrescribiendo cada uno de ellos:
```

**Review by realFranco — CHANGES_REQUESTED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by realFranco — APPROVED** [review]

File: pages.es/common/autoflake.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# autoflake
+
+> Herramienta para eliminar importaciones y variables no utilizadas en archivos con extensión y sintaxis válida en Python.
```
Comment (navarroaxel) [inline pages.es/common/autoflake.md:3]:
```suggestion
> Herramienta para eliminar importaciones y variables no utilizadas en archivos Python.
```

**Comment by kant** [comment]
[PR Labeler](https://github.com/tldr-pages/tldr/actions/workflows/labeler.yml)

**Review by blueskyson — APPROVED** [review]

**Comment by kbdharun** [comment]
@kant Can you check this page again and make any modifications if necessary?

File: pages.es/common/autoflake.md (hunk 7 above)
Comment (kant) [reply pages.es/common/autoflake.md]: Looks good to me. Thanks!

**Pull request closed without merging**